pulser-simulation/pulser_simulation/simresults.py

```diff
@@ -373,7 +381,7 @@ class CoherentResults(SimulationResults):
 
     def __init__(
```

Comment:
Isn't this one specific to Qutip? It should likely go to the `qutip_result.py` module, no?

Reply:
Just for sake of bundling together stuff which are related to Qutip. But it can also stay separate, as long as you mention in the comments that it is used in Qutip only (for backward compatibility).